vnext/Microsoft.ReactNative/Views/FrameworkElementViewManager.cpp

```diff
@@ -344,20 +345,62 @@ bool FrameworkElementViewManager::UpdateProperty(
           const std::string &innerName = pair.first;
           const auto &innerValue = pair.second;
 
-          if (innerName == "selected")
+          if (innerName == "selected") {
             states[static_cast<int32_t>(winrt::Microsoft::ReactNative::AccessibilityStates::Selected)] =
                 innerValue.AsBoolean();
-          else if (innerName == "disabled")
+            const auto prevState = DynamicAutomationProperties::GetAccessibilityStateSelected(element);
+            auto peer = xaml::Automation::Peers::FrameworkElementAutomationPeer::FromElement(element);
```

Comment:
this can be extracted out of the if and just used throughout every case as needed

Reply:
Extracted the automation peer bit since that stays the same, prevState depends on which value we're changing so I just renamed to make it more clear which previous value we're referring to in each case.